src/Symfony/Component/Cache/Tests/Adapter/CouchbaseCollectionAdapterTest.php

```diff
@@ -12,30 +12,25 @@
 namespace Symfony\Component\Cache\Tests\Adapter;
 
 use Psr\Cache\CacheItemPoolInterface;
-use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
 use Symfony\Component\Cache\Adapter\AbstractAdapter;
 use Symfony\Component\Cache\Adapter\CouchbaseCollectionAdapter;
 
 /**
  * @requires extension couchbase <4.0.0
  * @requires extension couchbase >=3.0.5
  *
- * @group legacy integration
+ * @group integration
  *
  * @author Antonio Jose Cerezo Aranda <aj.cerezo@gmail.com>
  */
 class CouchbaseCollectionAdapterTest extends AdapterTestCase
 {
-    use ExpectDeprecationTrait;
-
     protected $skippedTests = [
         'testClearPrefix' => 'Couchbase cannot clear by prefix',
     ];
 
     public function createCachePool($defaultLifetime = 0): CacheItemPoolInterface
     {
-        $this->expectDeprecation('Since symfony/cache 7.1: The "Symfony\Component\Cache\Adapter\CouchbaseBucketAdapter" class is deprecated, use "Symfony\Component\Cache\Adapter\CouchbaseCollectionAdapter" instead.');
```

Comment:
why removing this ?

Reply:
This seems to be a C&P mistake: The test covers the non-deprecated `CouchbaseCollectionAdapter`, but the deprecation is meant to be raised by the deprecated `CouchbaseBucketAdapter` class.

Reply:
I've extracted this change as #54979.